src/System.Runtime.Serialization.Formatters/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,3 +1,9 @@
 # Exposed publicly only in implementation for serialization compat
 TypesMustExist : Type 'System.Runtime.Serialization.SerializationEventHandler' does not exist in the reference but it does exist in the implementation.
 TypesMustExist : Type 'System.Runtime.Serialization.TypeLoadExceptionHolder' does not exist in the reference but it does exist in the implementation.
+
+MembersMustExist : Member 'System.Runtime.Serialization.SerializationInfo.DeserializationInProgress.get()' does not exist in the reference but it does exist in the implementation.
```

Comment:
NIT: Looks like this baseline has an extra space before the first MembersMustExist.

Reply:
intentional since the comment above doesn't necessarily apply to the ones that I added.